src/gf.c

```diff
@@ -2859,20 +2861,19 @@ STATIC_INLINE jl_value_t *verify_type(jl_value_t *v) JL_NOTSAFEPOINT
     return v;
 }
 
+STATIC_INLINE jl_value_t *invoke_codeinst(jl_value_t *F, jl_value_t **args, uint32_t nargs, jl_code_instance_t *codeinst)
+{
+    jl_callptr_t invoke = jl_atomic_load_acquire(&codeinst->invoke);
+    jl_value_t *res = invoke(F, args, nargs, codeinst);
+    return verify_type(res);
+}
+
 STATIC_INLINE jl_value_t *_jl_invoke(jl_value_t *F, jl_value_t **args, uint32_t nargs, jl_method_instance_t *mfunc, size_t world)
 {
     // manually inlined copy of jl_method_compiled
```

Comment:
This contradicts the comment now. Are you sure the code is still optimized?

Reply:
I had overlooked this comment. I simply removed the duplicate code, but it seems you were trying to optimize by manual inline. I haven't done any performance checks on my end, so it might be best to revert the changes.

Reply:
Should we add `FORCE_INLINE`/`STATIC_INLINE`there?